.travis.yml

```diff
@@ -19,19 +23,33 @@ env:
     - ZEPHYR_SDK_DOWNLOAD_FOLDER=https://nexus.zephyrproject.org/content/repositories/releases/org/zephyrproject/zephyr-sdk/$ZEPHYR_SDK_VERSION-i686
     - ZEPHYR_SDK_SETUP_BINARY=zephyr-sdk-$ZEPHYR_SDK_VERSION-i686-setup.run
     - ZEPHYR_SDK_DOWNLOAD_URL=$ZEPHYR_SDK_DOWNLOAD_FOLDER/$ZEPHYR_SDK_SETUP_BINARY
-  matrix:
-    - TARGET="zephyr"
-    - TARGET="extra"
-    - TARGET="linux"
-    - TARGET="ashell"
+matrix:
+  include:
+    - os: linux
+      env: TARGET="zephyr"
+    - os: linux
+      env: TARGET="extra"
+    - os: linux
+      env: TARGET="linux"
+    - os: linux
+      env: TARGET="ashell"
+    - os: osx
+      osx_image: xcode8.3
+      env: TARGET="linux"
```

Comment:
I'd like something different here to distinguish which VM is which in the Travis UI. So maybe add OS="osx" to the last one? The variable won't be used by we can at least read that then.